emoji/src/main/java/com/vanniktech/emoji/EmojiEditText.java

```diff
@@ -86,4 +87,25 @@ public final void setEmojiSizeRes(@DimenRes final int res) {
   public final void setEmojiSizeRes(@DimenRes final int res, final boolean shouldInvalidate) {
     setEmojiSize(getResources().getDimensionPixelSize(res), shouldInvalidate);
   }
+
+  /** Forces that only Emojis are allowed as input. */
+  public void forceEmojisOnly(final EmojiPopup emojiPopup) {
+    setOnFocusChangeListener(new ForceEmojisOnlyFocusChangeListener(emojiPopup));
```

Comment:
This doesn't do what the method name suggests, it shows the popup once the EditText is focused instead.

Reply:
So `showsEmojisOnly`. With this approach, there's no way to get to the normal keyboard though and insert normal characters.

Reply:
Ah no I get what this is supposed to do! I got confused since I thought this method should control if the `OnlyEmojisInputFilter` should be applied or not, but it's somethings else.

Maybe a name like `disableKeyboardInput` or something would be more clear?

Reply:
If we add this, we may want a method to revert this behavior e.g. remove the `OnFocusListener`.